doc/api/deprecations.md

```diff
@@ -2697,6 +2697,39 @@ The `repl` module exports a `_builtinLibs` property that contains an array with
 native modules. It was incomplete so far and instead it's better to rely upon
 `require('module').builtinModules`.
 
+<a id="DEP0143"></a>
```

Comment:
Realizing in retrospect taht this should have landed without an assigned deprecation number, this should happen at the time of release.

Reply:
FWIW this PR was using a placeholder originally, and I changed it to the deprecation number based on https://github.com/nodejs/node/pull/33430#issuecomment-629626771

Reply:
Deprecation codes are supposed to be assigned when a PR lands: https://github.com/nodejs/node/blob/master/doc/guides/releases.md#step-3-update-any-replaceme-and-dep00xx-tags-in-the-docs